tests/integration/test_installation.py

```diff
@@ -149,6 +149,14 @@ def test_running_real_remove_backup_groups_job(ws, sql_backend, new_installation
         ws.groups.get(ws_group_a.id)
 
 
+@retried(on=[NotFound, InvalidParameterValue, OperationFailed], timeout=timedelta(minutes=10))
+def test_repair_run_assessment_job(ws, new_installation, caplog):
+    install = new_installation()
+    install.run_workflow("assessment")
```

Comment:
It's easier to "break" the destroy database workflow, where you can inject failure by removing the database before starting the job

Reply:
used destroy database workflow to test the repair run in Integration test

Reply:
<img width="739" alt="Screenshot 2023-12-28 at 2 52 59 PM" src="https://github.com/databrickslabs/ucx/assets/127273819/0f54c3c7-3bcf-410e-824c-f79ca7063337">